src/ContentBody.css

```diff
@@ -178,4 +178,12 @@
     width: 200px;
     height: 300px;
     border-radius: 20px;
-}
\ No newline at end of file
+}
+.content-bottom1 img:hover ,.content-bottom3 img:hover {
+    transform: scale(1.2);
+    z-index: 1;
+  }
+  .content-bottom2 img:hover , .content-left img:hover , .content-right  img:hover{
```

Comment:
What is this🧐

Please read about css selector naming guidelines, you should do more with less nesting